dotcom-rendering/src/components/MostViewedRightWithAd.tsx

```diff
@@ -31,6 +33,9 @@ export const MostViewedRightWithAd = ({
 	const useDarkColourScheme =
 		format.design === ArticleDesign.Video ||
 		format.design === ArticleDesign.Audio;
+	const deeplyReadTestVariant = useDeeplyReadTestVariant();
```

Comment:
Unfortunately this won't work given that we're on the server here and this is a client-side test. The options as I see it is to make this component an Island or to update the height of this container with javascript.

This change can be reverted later when we remove the test and implement one of these three approaches. As it currently is, the Most Viewed component will be significant higher in the `deeply-read-and-most-viewed` variant. 

`deeply-read-and-most-viewed`
<img width="500" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/9574885/e56e4f67-017d-43ca-b06f-ee298cfbdda5">

`most-viewed-only`
<img width="500" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/9574885/e8e2ee1b-df9e-4607-bd5e-23b67fea1d47">